rich/_inspect.py

```diff
@@ -233,3 +233,38 @@ def _get_formatted_doc(self, object_: Any) -> Optional[str]:
         if not self.help:
             docs = _first_paragraph(docs)
         return escape_control_codes(docs)
+
+
+def object_types_mro(obj: Union[object, Type[Any]]) -> Tuple[type, ...]:
+    """Returns the MRO of an object's class, or of the object itself if it's a class."""
+    obj_type: Type[Any] = type(obj)
```

Comment:
Is this function strictly necessary? The call to `getattr` pretty much negates any benefit from typing. I'd be tempted to fold this function in to the function below.